docs/release-notes/8.0/8.0.35-30.0.md

```diff
@@ -0,0 +1,25 @@
+# Percona XtraBackup 8.0.35-30 (2023-11-)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Release highlights
+
+This release merges the MySQL 8.0.35 code base.
+
+## Bug fix
+
+* [PXB-3003](https://jira.percona.com/browse/PXB-3003): The `log_files_find_and_analyze` option had a potential race condition when the user listed the available files and tried to open them. In between these two operations, the server could delete the log files, causing the `log_files_find_and_analyze` option to return an error.
+
+* [PXB-3139](https://jira.percona.com/browse/PXB-3139): Fix for gcc-13 and clang-16 compilation issues.
+
+* [PXB-3147](https://jira.percona.com/browse/PXB-3147): Percona XtraBackup failed to execute the `DO innodb_redo_log_consumer_register("PXB");` query.
+
+* [PXB-3181](https://jira.percona.com/browse/PXB-3181): The use of the `innodb-use-native-aio=true` option resulted in printing duplicate timestamps. 
```

Comment:
I think there is one more bug fix from marce. @altmannmarcelo ? the partial download fix for xbcloud

Reply:
Please add 
https://jira.percona.com/browse/PXB-3173
https://jira.percona.com/browse/PXB-3141